config/config.go

```diff
@@ -133,6 +133,7 @@ type ServerConfig struct {
 
 	AlwaysCollectSystemData bool `ini:"always_collect_system_data"`
 	DisableCitusSchemaStats bool `ini:"disable_citus_schema_stats"`
+	DisableCitusIndexStats  bool `ini:"disable_citus_index_stats"`
```

Comment:
I was wondering whether we should instead make `disable_citus_schema_stats` have three options? `all`, `index`, `none` (with the default being `none`)

Though I guess technically you could enable index stats and not table stats, so maybe this approach works too? (at the cost of adding an additional config setting)

Reply:
That would be simpler. Filtering out table but not index stats is an unlikely use case, so I don't think we need to support that (and if we do, we could always allow `disable_citus_schema_stats=table`).

It would be a backward compatibility break, though. Or I guess we could map truthy values to "all" and falsey ones to "none"? Maybe I'll try that; that seems reasonable.

Reply:
Okay, did that. Our existing tests don't cover this, and it's probably not worth the CI cycles, but I tried adding various values of the setting to the `citus` integration test and it behaves as expected.